src/test/java/emissary/kff/KffQuincyFileTest.java

```diff
@@ -0,0 +1,40 @@
+package emissary.kff;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import emissary.test.core.UnitTest;
+import emissary.util.io.ResourceReader;
+import org.junit.Before;
+import org.junit.Test;
+
+public class KffQuincyFileTest extends UnitTest {
+
+    private static final String resourcePath = new ResourceReader()
+            .getResource("emissary/kff/test.dat").getPath();
+    private static KffQuincyFile kffQuincyFile;
+
+    @Override
+    @Before
+    public void setUp() throws Exception {
+        kffQuincyFile = new KffQuincyFile(resourcePath, "testFilter", KffFilter.FilterType.Unknown);
+    }
+
+    @Test
+    public void testCreation() {
+        assertEquals("MD5", kffQuincyFile.getPreferredAlgorithm());
```

Comment:
This should also assert recordLength is 16 as after the preferred alg, that's the reason for this object.